examples/public/global.css

```diff
@@ -64,3 +64,8 @@ button:not(:disabled):active {
 button:focus {
 	border-color: #666;
 }
+
+.cld-responsive{
```

Comment:
I see that code-wrapper i used for the testing but what are the other styling used for?

Reply:
global.css has default styles that come with the svelte app template, i added .cld-responsive